### PR TITLE
API2400 for storage resources

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
API2400 for storage resources

### Check List
- [ ] New functionality includes testing.
  - [x]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,70 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
+# [v1.7.0](unreleased)
+### Notes
+- This release supports API2200 minimally where we can use OneView API2400 with this SDK and ImageStreamer REST API version 2000 (I3S v5.40).
```
Comment (yuvirani) [inline CHANGELOG.md:5]: Change to "supports API2400 minimally" and i3s also

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,70 @@
...
+
+### Oneview Features supported
+- Enclosure
+- Enclosure Group
+- Ethernet Network
+- FC Network
+- FCOE Network
+- Interconnects
+- Interconnect Types
+- Network Set
```
Comment (yuvirani) [inline CHANGELOG.md:17]: Move below LIG

File: endpoints-support.md [hunk 3]
```diff
@@ -110,27 +110,27 @@
...
+|<sub>/rest/storage-volumes</sub>                                                             | POST     | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:         | :heavy_multiplication_x:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/storage-volumes/{id}</sub>                                                        | PUT      | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:         | :heavy_multiplication_x:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
```
Comment (yuvirani) [inline endpoints-support.md:124]: endpoint missing for DELETE

**Review by yuvirani — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (nabhajit-ray) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (nabhajit-ray) [reply CHANGELOG.md]: Done

**Pull request merged**